src/main/java/graphql/schema/GraphQLTypeUtil.java

```diff
@@ -1,26 +1,28 @@
 package graphql.schema;
 
-import graphql.Internal;
+import graphql.PublicApi;
 
-@Internal
+/**
+ * A utility class that helps work with {@link graphql.schema.GraphQLType}s
+ */
+@PublicApi
 public class GraphQLTypeUtil {
 
     /**
-     * This will get the unwrapped type name that includes the non null and list wrappers
-     * so it might be '[typeName!]'
+     * This will return the type in graphql AST format, eg [typeName!]!
      *
      * @param type the type in play
      *
-     * @return the unwrapped type name
+     * @return the type in graphql AST format, eg [typeName!]!
      */
-    public static String getUnwrappedTypeName(GraphQLType type) {
+    public static String toAst(GraphQLType type) {
```

Comment:
```suggestion
    public static String simplePrint(GraphQLType type) {
```
I feel like toAst is a bit unclear, because we actually don't return an Ast (it is not a Node or so). So I am thinking about print or simplePrint (to make it more more clear that it is not a full print compared to SchemaPrinter)